bazel/setup_clang.sh

```diff
@@ -20,7 +20,7 @@ build:clang --action_env=CXX=clang++
 build:clang --action_env=LD_LIBRARY_PATH=$(llvm-config --libdir)
 build:clang --test_env=LD_LIBRARY_PATH=$(llvm-config --libdir)
 
-build:clang-asan --action_env=ENVOY_UBSAN_VPTR=1
+build:clang-asan --action_env=ENVOY_UBSAN=1
```

Comment:
this is more WEE8_UBSAN now, the rest of files is still build with sanitize=undefined.

Reply:
Literally the very next line defines `-fsanitize=vptr,function`.

Reply:
with that in mind, should I leave it as `ENVOY_UBSAN_VPTR`?

Reply:
`-fsanitize=vptr,function` is only part of UBSAN, they explicitly disabled in `.bazelrc` but re-enabled here. The rest of UBSAN is still enabled regardless for the rest.

Reply:
Yes, my point is, this isn't enabled only for wee8, but for all targets, so I don't understand why are you suggesting that this should be called `WEE8_UBSAN` instead of `ENVOY_UBSAN`.

Reply:
Because the rest of UBSAN are enabled (e.g. alignment, null, pointer-overflow) without `clang.bazelrc`.

Reply:
Agreed, but that has nothing to do with wee8. Yes, those environment variables are only consumed by wee8 right now, at least among stuff in `envoyproxy/envoy`, and they basically expose which configuration is applied in Bazel. This is not at all wee8 specific.

We already have `ENVOY_ASAN`, `ENVOY_TSAN`, `ENVOY_MSAN` in `.bazelrc`, so I don't understand the bike-shedding here.

Reply:
So again, should I leave it as `ENVOY_UBSAN_VPTR`?

Reply:
yeah, or `ENVOY_UBSAN_CXX`